CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Metroid Dread
 
+- Added: 3 videos to logic database for a a diagonal bomb jump in Ghavoran, a single wall jump in Cataris, and a diffusion abuse trick in Artaria.
+- Changed: SWJ trick in Cataris Teleport to Artaria (Blue) now requires a slide jump.
```

Comment:
First line: There are two "a"s in a row. Remove one.
Second line: Move it to the Cataris section of the Changelog and unabbreviate SWJ to Single-wall Wall Jump